code/modules/library/skill_learning/skillchip.dm

```diff
@@ -23,6 +23,8 @@
 	var/removable = TRUE
 	/// How many skillslots this one takes
 	var/slot_cost = 1
+	/// If the chip can be implanted multiple times
+	var/allow_multiple = 0
```

Comment:
Use boolean defines (`FALSE` and `TRUE`) for boolean flags.
Or even better, use a bitflag here, something like `var/skillchip_flags = NONE` and `#define SKILLCHIP_ALLOWS_MULTIPLE (1<<0)`

Reply:
Changed it to the bitflag system, thanks to MrPerson and Nemvar for explaining them a bit more